action.yml

```diff
@@ -48,7 +48,7 @@ inputs:
       (Exactly one of `checks-yaml` and `checks-filename` are required.)
 
 runs:
-  using: node12
+  using: node16
```

Comment:
I may not end up making this change. I can't tell if `node16` is supported in `actions-runner-controller` yet.